### ISSUE TITLE
Language JSON-config files

### ISSUE THREAD

**Issue by casperlundberg** [PepperonIT/blockly-site#56]
It would be nice to have all the language determined strings in JSON-files with each file representing a language. Just to have all the translations in the same place and also all strings so it would be much easier to add a new language and add new blocks and/or elements of the site that need translations. 

Should we fix this before we deploy it on LMS or is it a "future work" worth mentioning in the report?

**Comment by noorbhtiti** [comment]
Maybe future work ?? I can do it, but I don't see the meaning of making that now. I would rather put my time to do other things like studying for my exam  @casperlundberg

**Comment by casperlundberg** [comment]
Agree, I'd rather talk to Peter and do this issue and some of the other after the exams as a hobby.
